pallets/ah-migrator/src/bounties.rs

```diff
@@ -154,7 +156,15 @@ impl<T: Config> crate::types::AhMigrationCheck for BountiesMigrator<T> {
 			"Bounty descriptions map value on Asset Hub should match RC value"
 		);
 
+		// Assert storage 'Bounties::BountyApprovals::ah_post::length'
```

Comment:
We could probably change the script to always mark the length requirement as checked when the `correct` requirement is fulfilled.